integration/graphql/src/main/java/com/blazebit/persistence/integration/graphql/GraphQLEntityViewSupportFactory.java

```diff
@@ -1156,7 +1160,18 @@ protected GraphQLOutputType getScalarType(GraphQLSchema.Builder schemaBuilder, C
         return new GraphQLTypeReference(typeName);
     }
 
+    /**
+     * Either determines the nullability of a field from the annotations present on the entity view or if no overrides
+     * present, derive it from the hibernate entity referenced by the view.
+     */
     protected boolean isNotNull(SingularAttribute<?, ?> attribute, EntityMetamodel entityMetamodel) {
+        boolean isEntityView = attribute.getDeclaringType().getJavaType().isAnnotationPresent(EntityView.class);
```

Comment:
This is unnecessary, because this is only called for entity view attributes. Please remove this part.

Reply:
done